### PR TITLE
Small rat audit

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Bugfixes "Fix rat aggression"

#### Purpose of change

Rats are weird. Black rats are aggressive until you get close, which often activates safe mode making it a pain to do stuff like hauling or autodriving. They also have a few weird flags, like stumbling. Brown rats (renamed from sewer rats) also had really shitty day vision for no reason, and could upgrade.

#### Describe the solution

Fixes all of that.

#### Describe alternatives you've considered

Playing nothing but DDotD so no rats.

#### Testing

Till I bled

#### Additional context
http://www.ratbehavior.org/RatVision.htm Rats kinda suck at seeing stuff

### CODE REVIEW

**Comment by Nebnis** [comment]
Rats have very poor eye sight during the day https://spca.bc.ca/wp-content/uploads/Bark-spring-2011-rat-senses-article.pdf

**Comment by Holli-Git** [comment]
> Rats have very poor eye sight during the day https://spca.bc.ca/wp-content/uploads/Bark-spring-2011-rat-senses-article.pdf

Does dropping it to 10 or 15 for day and 10 for night sound good?

**Comment by Nebnis** [comment]
Well, i will probably let others have more input in my opinion both rats should be 1~2 tiles during the day.  Since 1 tile is 3 meters, that already seems far too much for a rat to see during the day since sunlight is supposed to physically hurt them. So 1 or 2 tiles should be enough

Could give them instead very good hearing and smell and make them run away from those things with a sounds fear flag.

**Comment by Holli-Git** [comment]
> Well, i will probably let others have more input in my opinion both rats should be 1~2 tiles during the day. Since 1 tile is 3 meters, that already seems far too much for a rat to see during the day since sunlight is supposed to physically hurt them. So 1 or 2 tiles should be enough
> 
> Could give them instead very good hearing and smell and make them run away from those things with a sounds fear flag.

What? 1 tile is 1 meter (usually)

File: data/json/monsters/rodentkin.json [hunk 1]
```diff
@@ -34,8 +34,8 @@
     "melee_dice_sides": 1,
     "melee_damage": [ { "damage_type": "cut", "amount": 1 } ],
     "dodge": 2,
-    "vision_day": 30,
-    "vision_night": 15,
+    "vision_day": 15,
+    "vision_night": 10,
```
Comment (Karol1223) [inline data/json/monsters/rodentkin.json:38]: Why's this guy got better vision than the other albino lab rat?

**Review by Karol1223 — CHANGES_REQUESTED** [review]
Overall the sights seem fine to me, if maybe a little low still. Keep in mind our 'sight' isn't only sight, it's all the senses that give the animal (in this specific range) perception ***adequate to that of sight***. I have a hard time explaining what I mean, but look at Headless Horrors - they don't see shit, but they do 'see' in the game, because they have their feelers, which mean their perception of their surroundings is (albeit on a smaller range, hence the JSON stats) adequate that sight would provide them.

Rodents don't have *terribly* good senses. There's a population of field mice that lives on my regular path to college, and they typically start scurrying away only when I'm a couple meters away. But 1-2 tiles would be insultingly low given how our tile sizes (and speeds!) are.

Personally I'd do like 8-10 day and 5-7 night visions, but I think you're in the right ballpark with most here anyway.

File: data/json/monsters/rodentkin.json (hunk 1 above)
Comment (Holli-Git) [reply data/json/monsters/rodentkin.json:38]: The buggers are mutant lil buggers, felt it made sense if they could see a bit better

Comment (Karol1223) [reply data/json/monsters/rodentkin.json:38]: > The buggers are mutant lil buggers, felt it made sense if they could see a bit better

Nah they're regular rats high on purifier, made specifically to be exact replicas that are able to upgrade.

File: data/json/monsters/mammal.json [hunk 2]
```diff
@@ -2320,8 +2319,8 @@
   {
     "id": "mon_sewer_rat",
     "type": "MONSTER",
-    "name": { "str": "sewer rat" },
-    "description": "A worm-tailed rodent with long whiskers and beady eyes.  The way it squeaks makes it sound… hungry.",
+    "name": { "str": "brown rat" },
+    "description": "The brown rat is worm-tailed rodent with long whiskers and beady eyes, larger than its cousin, the black rat.  Widespread where ever humans are (save for Antarctica and Alberta), they live off of foraging and will eat whatever they can find.",
```
Comment (Karol1223) [inline data/json/monsters/mammal.json:2323]:
```suggestion
    "description": "The brown rat is a worm-tailed rodent with long whiskers and beady eyes, larger than its cousin, the black rat.  Widespread mostly wherever humans are, they live off of foraging and will eat whatever they can find.",
```

**Review by Karol1223 — CHANGES_REQUESTED** [review]

File: data/json/monsters/rodentkin.json (hunk 1 above)
Comment (Holli-Git) [reply data/json/monsters/rodentkin.json:38]: Alright, I'll change it then

**Pull request merged**